wallet-api/src/Wallet/Emulator/Types.hs

```diff
@@ -196,6 +198,10 @@ handleNotifications = mapM_ (updateState >=> runTriggers)  where
     update t = over addressMap (AM.updateAddresses t)
 
 instance WalletAPI MockWallet where
+    validationInterval = do
+        s <- slot
+        pure (s, s + 1000)
```

Comment:
I do like to have all magic constants as named values, so it's easier to see what they are for and grep for them if you want to change them. Something like `mockDefaultValidationInterval`.